bobtail-dev/bobtail
# Integrate Reactive SVG functionality #14

This streamlines the patch from #14 by simply adding a list of SVG tags to the existing HTML tags.  I also added tests to verify they are added correctly, and ran the existing tests to make sure they didn't break anything.

To verify:
- create SVG elements (e.g., svg, rect)
- see that they are generated and display properly
